client/src/components/Card/Card.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./Card.css";
+
+function Card() {
+  return (
+    <div className="card">
+      <div>
+        <img src="http://placekitten.com/g/200/300" className="img"></img>
```

Comment:
Can you think of why this image might be distorted in your site?

Reply:
I think it is okay, I applied mods in css file